packages/react-components/react-card/src/components/Card/Card.cy.tsx

```diff
@@ -240,4 +260,115 @@ describe('Card', () => {
       });
     });
   });
+
+  describe('selectable', () => {
+    it('should not be selectable by default', () => {
+      mountFluent(<CardSample />);
+
+      cy.get(`.${cardClassNames.select}`).should('not.exist');
+    });
+
+    it('should role="checkbox" when selectable - selected prop', () => {
+      mountFluent(<CardSample selected />);
+
+      cy.get('#card').should('have.attr', 'role', 'checkbox');
+    });
+
+    it('should role="checkbox" when selectable - defaultSelected prop', () => {
+      mountFluent(<CardSample defaultSelected />);
+
+      cy.get('#card').should('have.attr', 'role', 'checkbox');
+    });
+
+    it('should role="checkbox" when selectable - onCardSelect prop', () => {
+      const Example = () => {
+        const onCardSelect = React.useCallback(() => null, []);
+
+        return <CardSample onCardSelect={onCardSelect} />;
+      };
+
+      mountFluent(<Example />);
+
+      cy.get('#card').should('have.attr', 'role', 'checkbox');
+    });
+
+    it('should have internal checkbox when selectable - onCardSelect prop', () => {
```

Comment:
As I understand the implementation, would this make more sense?

```suggestion
    it('should not add checkbox role to the root when a custom slot is provided - onCardSelect prop', () => {
```